### DISCUSSION TITLE
Nightvision Goggles

### DISCUSSION THREAD

**Discussion by Serrobii** [FakeFishGames/Barotrauma#8482] [Ideas and suggestions]
I need something that I can attach to me that creates a big area of visibility, but flashlights are pretty meh in general, especially when moving underwater.

It could have the drawback of having diving suit visibility so it's somewhat balanced.

**Comment by GoodGuyJimmy** [comment]
I guess that could be one of those talent-unlocked crafting recipes, kinda like the thermal goggles.
